internal/provider/data_source_packer_image_iteration.go

```diff
@@ -0,0 +1,210 @@
+package provider
+
+import (
+	"context"
+	"log"
+	"time"
+
+	packermodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-packer-service/preview/2021-04-30/models"
+	sharedmodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-shared/v1/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-hcp/internal/clients"
+)
+
+// a minute sounds like a lot already since this would be mainly pulling data.
```

Comment:
[nit] could drop this or leave a TODO.

Reply:
Got it, yeah dropping it, I think 1 min is okay for now 🙂 
```suggestion
```